server/src/DBTablesLastInfo.cc

```diff
@@ -207,9 +207,8 @@ HatoholError DBTablesLastInfo::getLastInfoList(LastInfoDefList &lastInfoDefList,
 	getDBAgent().runTransaction(arg);
 
 	const ItemGroupList &grpList = arg.dataTable->getItemGroupList();
-	ItemGroupListConstIterator itemGrpItr = grpList.begin();
-	for (; itemGrpItr != grpList.end(); ++itemGrpItr) {
-		ItemGroupStream itemGroupStream(*itemGrpItr);
+	for (auto item : grpList) {
```

Comment:
Oh, I see. Yes, `range_declalation`is not a single item. They contain multiple values.